pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -122,6 +122,19 @@ type TresorCASpec struct {
 // can be (Active, Passive).
 type MeshRootCertificateIntent string
 
+const (
+	// ActiveIntent means the settings and certificate provider in this MRC are used for signing and
+	// validating certificates.
+	ActiveIntent MeshRootCertificateIntent = "active"
```

Comment:
I like these constants here but we should clean up the ones not used in https://github.com/openservicemesh/osm/blob/main/pkg/constants/constants.go#L203

Reply:
Removed